src/tables/cpal.rs

```diff
@@ -20,7 +20,7 @@ impl<'a> Table<'a> {
         let mut s = Stream::new(data);
 
         let version = s.read::<u16>()?;
-        if version != 0 {
+        if version > 1 {
```

Comment:
v0 and v1 are the same?

Reply:
The first part of the table should be the same. That's why we have an early return in the function if version is 0.

Reply:
Do we support v1? Does it even have a useful info for us?